app/src/test/java/nl/tudelft/bbw/APITest.java

```diff
@@ -47,13 +49,21 @@
     @Before
     public final void setUp() throws HashException, BlockAlreadyExistsException {
         EdDSAPrivateKey privateKey = ED25519.generatePrivateKey();
-        owner = new User("Jeff", "iban", ED25519.getPublicKey(privateKey));
+        owner = new Acquaintance("Jeff", "iban", ED25519.getPublicKey(privateKey), new ArrayList<List<Block>>());
```

Comment:
Actually when initialize the Acquaintance, there should be a database with at least a genesis block of this contact available. But since this is a test Acquaintance, we do not have any predefined database so to make an Acquaintance so I filled an empty multichain in. But if you look further to the next codes lines, the multichain is set to the one with a genesis block like it's supposed to be.